licenses/MADlib.txt

```diff
@@ -1,10 +0,0 @@
-Portions of this software Copyright (c) 2010-2013 by EMC Corporation.  All rights reserved.
```

Comment:
Thanks, Roman. 
Symlink would be the best option if we have to keep the file. 

Alternatively, we can change the `"${CMAKE_SOURCE_DIR}/licenses/MADlib.txt"` in `deploy/PackageMaker/CMakeLists.txt` to `"${CMAKE_SOURCE_DIR}/LICENSE"` and remove this file.